plane/code/.env.example

```diff
@@ -10,9 +10,9 @@ BEAT_WORKER_REPLICAS=1
 LIVE_REPLICAS=1
 
 NGINX_PORT=80
-WEB_URL=http://${APP_DOMAIN}
```

Comment:
this can be left as it was, because it's just a env interpolation based on the value of APP_DOMAIN, which will use PRIMARY_DOMAIN